lib/circlex/emulator/api/core/bank_accounts_api.ex

```diff
@@ -22,6 +23,14 @@ defmodule Circlex.Emulator.Api.Core.BankAccountsApi do
     end
   end
 
+  # https://developers.circle.com/reference/payments-bank-accounts-wires-get-id-instructions
+  @route "/wires/:bank_account_id/instructions"
+  def get_wire_instructions(%{bank_account_id: bank_account_id}) do
+    with {:ok, bank_account_sending_wire} <- BankAccountState.get_bank_account(bank_account_id) do
+      {:ok, WireInstructions.serialize(bank_account_sending_wire)}
```

Comment:
oh uh, this isn't really right. The instructions are always going to be Circles bank, with a dynamic Account Number. The BankAccount here is the customer's bank

Reply:
yup, but the [API](https://developers.circle.com/reference/payments-bank-accounts-wires-get-id-instructions) still takes in the customer's bank account to return the virtual account number corresponding to that account.

what I'm actually doing here is taking in the VAN and trackingRef stored in `Struct.BankAccount` to serialize a `WireInstructions` struct like this--
<img width="466" alt="Screen Shot 2022-08-12 at 3 56 23 PM" src="https://user-images.githubusercontent.com/21118039/184433476-1d4286f0-510d-473d-9d81-5aacc63b1ae2.png">

Reply:
okay, so it takes the argument as the bank account, and returns something of different shape. I think most of these are serialize functions are returning an object of the same shape, just going from map -> struct or struct -> map

Reply:
I guess the interface is "thing with virtual account number and tracking ref keys" ? That seems okay.